crawler/attribution/entrypoints/blob-url.js

```diff
@@ -0,0 +1,8 @@
+{
+    const b = new Blob([`console.log("${document.currentScript.src}", navigator.userAgent)`], { type: 'text/javascript' });
```

Comment:
I think we should be addResult-ing here no?

Reply:
Yup, you're right. Changed it